src/playground/Insights/ui/separator.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+
+export const Separator: React.FC = () => {
+  return (
+    <div
+      className="my-4 h-[1px] w-full"
+      style={{
+        backgroundImage:
+          "repeating-linear-gradient(to right, hsl(var(--secondary-intermediate)) 0, hsl(var(--secondary-intermediate)) 3px, transparent 3px, transparent 7px)",
```

Comment:
Can't this be done with a `border-dashed`?

Reply:
The think is that border dashed looks like - - - - - and the design is more like _ _ _ _ _ so I had to do this workaround.

Reply:
Not sure I’m following, do you mean the line is thicker in the design? If that's the case, I'm pretty sure we can find a happy medium and keep the separator simpler (even though I do like the workaround!).

Reply:
Sorry, I thought I did a great job explaining that, but evidently I didn't. The default dashed line in css is very different to the one in Figma
![image](https://github.com/user-attachments/assets/fcfccf75-70cf-44a4-a3f1-fd79bf75996a)
![image](https://github.com/user-attachments/assets/1d65f812-ce48-4182-92f2-5e3011f2639e)

Reply:
Oh, gotcha! You did a fantastic job explaining it, just a classic Monday for me 😅. Still wondering if we're overcomplicating things just for a separator, but let's roll with it!

Reply:
This is super easy to rollback and won't create any dependencies, so we can move forward with it and change it afterwards if we decide it.